ddsc/cmdparser.py

```diff
@@ -149,6 +158,17 @@ def _add_auth_role_arg(arg_parser):
                             help="Specifies authorization role for the user ('project_admin').",
                             default='project_admin')
 
+def _add_copy_project_arg(arg_parser):
```

Comment:
Why is this prefixed with a `_` when the other functions have had their `_` prefixes removed?